lib/cagen.py

```diff
@@ -359,6 +360,8 @@ def _get_hostname():
     Returns the hostname of the current system, returns None if it can't
     get the hostname. Stolen from osg-test
     """
+    if "OSG-FDQN" in os.environ:
+        return os.getenv("OSG-FDQN")
```

Comment:
```suggestion
    try:
        return os.getenv('OSG_FQDN')
    except KeyError:
        pass
```

A `try` block avoids two environment lookups and is more pythonic! Also, replacing the dash with an underscore (https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html)